sharedbuffers/mapped_struct.py

```diff
@@ -492,7 +497,7 @@ def is_wrapped_key(obj):
 @cython.inline
 def get_wrapped_key(obj):
     # the key for the unwrapped value
-    if six.PY2:
+    if python2:
         if isinstance(obj, tuple):
             return obj[1]
         elif isinstance(obj, int):
```

Comment:
Again, I prefer else for exclusive paths:

``` python
if python2:
   # py2 verison
else:
   # py3 verison
```

Instead of

``` python
if python3:
    # py3 version
# py2 version
```

It's just so much more readable in the first form.